resources/views/forum/topics/_post_info.blade.php

```diff
@@ -65,6 +65,11 @@ class="forum__user-flag forum__user-flag--country"
         </div>
 
         <div class="forum-post__info-panel-extra-bottom">
+            <div class="forum-post__info-row">
+                <a class="forum-post__info-link" href="{{ route("users.posts", $user) }}">
+                    {{ trans_choice('forum.post.info.post_count', number_format($user->user_posts)) }}
```

Comment:
`number_format` shouldn't be used here because the rule might be checked incorrectly (e.g. numbers between 1000 and 1999 will use singular form for English).

Reply:
should I just leave it un-formatted for now then? also, this pattern is used in a few other places

Reply:
yes, I'll fix it later.